Presentation.Web/app/helpers/ExcelExportHelper.ts

```diff
@@ -31,5 +32,61 @@
             }
             return "";
         }
+
+        static renderProjectStatusColor(status: Models.ItProject.IItProjectStatusUpdate[]) {
+            if (status.length > 0) {
+                var latestStatus = status[0];
+                var statusTime = latestStatus.TimeStatus;
+                var statusQuality = latestStatus.QualityStatus;
+                var statusResources = latestStatus.ResourcesStatus;
+                if (latestStatus.IsCombined) {
+                    return latestStatus.CombinedStatus;
+                } else {
+                    /* If no combined status exists, take the lowest status from the splitted status */
+                    if (statusTime === "Red" || statusQuality === "Red" || statusResources === "Red") {
+                        return "Rød";
+                    } else if (statusTime === "Yellow" || statusQuality === "Yellow" || statusResources === "Yellow") {
+                        return "Gul";
+                    } else if (statusTime === "Green" || statusQuality === "Green" || statusResources === "Green") {
+                        return "Grøn";
+                    } else {
+                        return "Hvid";
+                    }
+                }
+            } else {
+                return "";
+            }
+        }
+
+        static renderDate(dataItem: IItProjectInactiveOverview, moment: moment.MomentStatic) {
+
+            if (!dataItem.CurrentPhaseObj || !dataItem.CurrentPhaseObj.StartDate) {
```

Comment:
Den er lavet forkert. Den renderer jo kun en dato for en bestemt overview item og kun start date.. det er skidt